web/router.ex

```diff
@@ -35,6 +35,11 @@ defmodule Healthlocker.Router do
     put "/account/update", AccountController, :update
     get "/account/consent", AccountController, :consent
     put "/account/update-consent", AccountController, :update_consent
+    get "/account/security", AccountController, :security
+    get "/account/edit-security", AccountController, :edit_security
```

Comment:
You can use [`resources`](http://www.phoenixframework.org/docs/routing#section-resources) to let Phoenix generate all the routes for you. (`mix phoenix.routes` is super useful for testing this)

If there's a reason to use a specific route, then it might be a smell that we haven't modelled it correctly.